cellpack/autopack/ingredient/Ingredient.py

```diff
@@ -2913,12 +2999,16 @@ def jitter_place(
                     # found a collision, break this loop
                     break
                 else:
-                    insidePoints = self.merge_place_results(
-                        new_inside_points, insidePoints
-                    )
-                    newDistPoints = self.merge_place_results(
-                        new_dist_points, newDistPoints
-                    )
+                    if len(new_inside_points):
+                        insidePoints = self.merge_place_results(
+                            new_inside_points,
+                            insidePoints,
+                        )
+                    if len(new_dist_points):
```

Comment:
we talked about moving these `len` checks into `merge_place_results` so it doesn't have to be added each time the function is called.

Reply:
Removed these checks. They don't seem to be necessary since `merge_place_results` only loops if there are any new points to be added.